server/test/handlers/inputSchedule.test.js

```diff
@@ -0,0 +1,100 @@
+const workerTimeslotHandler = require('../../models/handlers/workerTimeslot');
```

Comment:
Please note that the handler file is called `timeslot` rather than `workerTimeslot` so I would suggest adjusting the file name here to fix the failing tests.

Reply:
Thank you @catai9 for your comment! I've made the changes accordingly.

Reply:
Did you remember to push? I still see the previous code referring to workerTimeslot.